src/Common/src/System/IO/PathInternal.Windows.cs

```diff
@@ -185,6 +181,7 @@ internal static bool IsDevice(StringBuilder path)
         /// <summary>
         /// Returns true if the path uses the extended syntax (\\?\)
         /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This PR adds 7 or so uses of AggressiveInlining.  If they're really all necessary, please follow-up with the JIT team to understand why.  Maybe there's something different we need to be doing in our inlining heuristics?  Maybe there's something specific to these cases that's giving it a headache? Etc.  Some of these are incredibly simple, so if they're actually not getting inlined automatically, that's surprising and a bit concerning, at least to me.  You might look at the ETW events generated by the JIT to understand exactly where and where not they're getting inlined.